README.md

```diff
@@ -1,3 +1,2 @@
 # CACAO2021
 
-A simulation of the cacao market for the CODEVSI course.
```

Comment:
You are changing files outside of your java package. You can see the files that you are changing in the "Files changed" tab of your pull request on the github website. This list must contain only files in your package.

You can replace them with the version from the main repository. To do that, right click on these files or folders -> Replace with -> Branch, Tag or Reference -> Remote tracking -> select the main repository -> Replace. If the files or folders do not exist in the main repository, you can simply delete them. Then commit and push to your repository. Don't close this pull request, it will be automatically updated.

Reply:
I did what you said, it's ok now ?

Reply:
No. You can see the files that you are changing in the "Files changed" tab of your pull request on the github website. This list must contain only files in your package.